servers/src/main/java/tachyon/master/file/meta/InodeDirectoryIdGenerator.java

```diff
@@ -57,10 +57,17 @@ synchronized long getNewDirectoryId() {
 
   @Override
   public synchronized JournalEntry toJournalEntry() {
-    return new InodeDirectoryIdGeneratorEntry(mContainerId, mSequenceNumber);
+    InodeDirectoryIdGeneratorEntry inodeDirectoryIdGenerator =
+        InodeDirectoryIdGeneratorEntry.newBuilder()
+        .setContainerId(mContainerId)
+        .setSequenceNumber(mSequenceNumber)
+        .build();
+    return JournalEntry.newBuilder()
+        .setInodeDirectoryIdGenerator(inodeDirectoryIdGenerator)
+        .build();
   }
 
-  public void fromJournalEntry(InodeDirectoryIdGeneratorEntry entry) {
+  public void initFromJournalEntry(InodeDirectoryIdGeneratorEntry entry) {
```

Comment:
noticed that this is non-static while others are all static. maybe my comment doesn't make sense :)